config/config.go

```diff
@@ -31,6 +31,14 @@ func GetURLOrBail(urlEnvVar string) *url.URL {
 	return parsedUrl
 }
 
+func GetOrDefault(envVar string, defaultValue string) string {
+	envValue := os.Getenv(envVar)
```

Comment:
technically you can set the value of the env var to the empty string, e.g. `USER=''`. You can distinguish between those cases with `LookupEnv` if you want. I think it's okay as is